jstz_api/src/http/header.rs

```diff
@@ -103,27 +99,21 @@ impl Headers {
     ///  - [WHATWG specification][spec]
     ///
     /// [spec] https://fetch.spec.whatwg.org/#dom-headers-get
-    pub fn get(&self, name: &str) -> JsResult<Option<Header>> {
+    pub fn get(&self, name: &str) -> JsResult<Option<String>> {
         let name = str_to_header_name(name)?;
-        let mut values = self.headers.get_all(name).into_iter();
-        match values.size_hint() {
-            (0, _) => Ok(None),
-            (1, Some(1)) => {
-                let header = values.next().expect("Expect 1 header");
-
-                Ok(Some(Header::Single(header_value_to_js_string(header)?)))
-            }
-            (1, None) => {
-                let values = values
-                    .map(header_value_to_js_string)
-                    .collect::<JsResult<Vec<JsString>>>()?;
+        let headers = collect_headers(self.headers.get_all(name))?;
 
-                Ok(Some(Header::Multiple(values)))
-            }
-            _ => todo!(),
+        match headers.len() {
+            0 => Ok(None),
+            1 => Ok(Some(headers.get(0).expect("infallible").into())),
+            _ => Ok(Some(headers.join(", "))),
         }
```

Comment:
I think this is semantically the same:
```suggestion
        match headers.len() {
            0 => Ok(None),
            _ => Ok(Some(headers.join(", "))),
        }
```

Reply:
For collecting headers, I think it would still be useful to have a `Header` struct:
```rust
pub struct Header {
  pub headers: Vec<String> // non-empty
}

impl Header {
  pub fn try_from_iter<T>(iter: T) -> JsResult<Self> where T: IntoIterator<Item = A> {
    // Code from `collect_headers`
  }
}

// Default conversion by joining w/ ", "
impl IntoJs for Header {
  ...
}
```